frontend_server/package_incremental.py

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env python
```

Comment:
Ensure this file has the execute bit set.

Reply:
Fixed